src/main/java/eu/enexa/sparql/SparqlBasedMetadataManager.java

```diff
@@ -134,4 +136,20 @@ public String getContainerName(String experimentIri, String instanceIRI) {
         return null;
     }
 
+    @Override
+    public List<String> getAllContainersName(String experimentIri) {
```

Comment:
Shouldn't it be called `getAllContainerNames`?

Reply:
you right :D